lib/pattern.js

```diff
@@ -0,0 +1,2307 @@
+(function (global, factory) {
```

Comment:
Er werden niet alleen wijzigingen uitgevoerd in de code van Webuniversum, maar ook in het de achterliggende depenency Cleave.js. Samen te bekijken of we kunnen bereiken wat we willen zonder wijzigingen, anders kunnen we best een PR voorzien bij Cleave.js met onze uitbreiding en via Webuniversum een update + wijziging aanvragen.

Reply:
Verder besproken met @Pieter-Beckers wat de nodige acties nog zijn, voorlopig kan deze PR gemerged worden.

Reply:
PR aangemaakt: https://github.com/nosir/cleave.js/pull/643